financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/FinancialConnectionsPlaygroundViewModel.kt

```diff
@@ -278,9 +255,18 @@ class FinancialConnectionsPlaygroundViewModel(
         true,
         emptySet()
     )
+
+    fun onSettingsChanged(playgroundSettings: PlaygroundSettings) {
+        _state.update {
+            it.copy(
+                settings = playgroundSettings,
+            )
+        }
+    }
 }
 
-enum class Merchant(val flow: String) {
+@Suppress("Unused")
```

Comment:
How come this is unused?

Reply:
the actual values are (individually) unused. We just iterate through the enum values.